e2etests/tests/graceful_restart.go

```diff
@@ -68,19 +72,25 @@ var _ = ginkgo.Describe("Establish BGP session with EnableGracefulRestart", func
 			dump.K8sInfo(testName, reporter)
 			dump.BGPInfo(testName, infra.FRRContainers, cs)
 		}
+
+		err := cleanup(updater)
+		Expect(err).NotTo(HaveOccurred(), "cleanup config in API and infra containers")
 	})
 
 	ginkgo.Context("When restarting the frrk8s deamon pods", func() {
+		ginkgo.DescribeTable("routes are maintained", func(ipFam ipfamily.Family, prefix, learnRoute string) {
```

Comment:
can we clarify that we are checking routes both sides? Would be a single test checking each side clearer?

Reply:
I will extend the description.

> Would be a single test checking each side clearer?

Not sure do you mean two tests? Given that test talkes 2minutes I would not add another instance of it for time reasons. Also checking in two ` consistently ` is not working because I must make sure that routes always exists.